asistente_ladm_col/gui/change_detection/dlg_change_detection_settings.py

```diff
@@ -266,29 +279,31 @@ def show_message_clean_workspace(self, message):
         msg.addButton(QPushButton(QCoreApplication.translate("ChangeDetectionSettingsDialog", "Yes, remove layers")), QMessageBox.YesRole)
         msg.addButton(QPushButton(QCoreApplication.translate("ChangeDetectionSettingsDialog", "No, don't remove")), QMessageBox.NoRole)
         msg.addButton(QPushButton(QCoreApplication.translate("ChangeDetectionSettingsDialog", "Cancel")), QMessageBox.RejectRole)
-        reply = msg.exec_()
+        msg.exec_()
 
-        if reply == QMessageBox.Yes:
+        # Get reply when use custom buttons
+        # https://stackoverflow.com/questions/25101171/what-do-the-different-qmessagebox-roles-mean
+        reply = msg.buttonRole(msg.clickedButton())
+
+        if reply == QMessageBox.YesRole:
             QgsProject.instance().layerTreeRoot().removeAllChildren()
             self.close_dialog()
-        elif reply == QMessageBox.No:
+        elif reply == QMessageBox.NoRole:
             self.close_dialog()
-        elif reply == QMessageBox.Cancel:
+        elif reply == QMessageBox.RejectRole:
             pass  # Continue config db connections
```

Comment:
We need to tell the users (via message bar) what happened after closing this dialog. E.g., "You have configured both Collected and Supplies database connections, now you can proceed to detect changes."

Reply:
Very important,it  has been included.